builtin/logical/database/rotation_test.go

```diff
@@ -709,6 +709,10 @@ func TestBackend_StaticRole_Rotations_PostgreSQL(t *testing.T) {
 }
 
 func TestBackend_StaticRole_Rotations_MongoDB(t *testing.T) {
+	if v := os.Getenv("GOARCH"); v == "386" {
```

Comment:
I think you can use `runtime.GOARCH` more reliably.

I am guessing the only 32-bit platform we test on is 386, but Go supports others, notably `arm`.

Would it be safer to disable the plugin altogether when on a 32-bit arch?

Reply:
Thanks for the input!

> Would it be safer to disable the plugin altogether when on a 32-bit arch?

Yes, I think so. Would that entail adding a check for `runtime.GOARCH` in the plugin and exiting if it is 32bit?

Reply:
That, or maybe checking the size of `uintptr`?

Reply:
looks like this may have been fixed in 1.8 version of the driver https://jira.mongodb.org/browse/GODRIVER-2236. Maybe I will try upgrading